packages/firestore/src/remote/serializer.ts

```diff
@@ -148,46 +146,33 @@ export class JsonProtoSerializer {
    * our generated proto interfaces say Int32Value must be. But GRPC actually
    * expects a { value: <number> } struct.
    */
-  private toInt32Value(val: number | null): number | null {
-    if (this.options.useProto3Json || typeUtils.isNullOrUndefined(val)) {
+  private toInt32Value(val: number | null): number | { value: number } | null {
```

Comment:
Optional: `toInt32Value` doesn't really do much to indicate that it's converting to a proto message. What do you think of renaming these (and related items) to names like `toInt32Proto`?

The Android/iOS convention of naming these `encode`/`decode` helps somewhat.

Reply:
I renamed this instance to `toInt32Proto` but did not convert the methods to the encode/decode pattern. I am personally also confused by the naming in the file and constantly try to use the wrong methods (`toValue` instead of `fromValue` etc)